### ISSUE TITLE
TODO: FRONT END

### ISSUE THREAD

**Issue by rspadim** [nfephp-org/sped-emissor#7]
PARTE DE FRONT END, DEIXEI COMO 9 PQ SEM A INTERFACE/API REST, NÃO TEM MUITO SENTIDO INTERLIGAR, MAS JA PODEMOS DESENVOLVER O FRONT!

9) CRIAR TELAS!
9.0) login: "USUARIO + SENHA + BOTÃO LOGIN" (a principio não precisa esqueci a senha, depois pensamos nisso, pois precisaria de uma interface de email, q não acho q é o foco ainda)
9.1) cadastro de emitentes
9.2) consulta / gerenciamento de documetnso fiscais
9.3) consulta / gerenciamento de eventos/cancelamento
9.4) consulta / gerenciamento de inutilizações

9.2 / 9.3 / 9.4) precisamos de uma tela simples para mostrar erros do processo executado em "importar/validar/assinar/transmitir/cancelar/intuilizar/registrar carta de correção(evento)/consultar sefaz"
9.5) tela para improtar do sistema de arquivo, ou receber upload (por http/html) - isto é para quem utiliza o emissor por http, ou não esta com uma interface de linha de comando (php-cli)

9.6) telas de edição de documetnso fiscais:
9.6.1) editar documentos (igual ao emissor do sefaz, mas isto é por ultimo, famos focar no backend, e quando estivermos com algo pronto integramos com o front, esta é a parte mais pesada do front)
9.6.1.1) primeiro NFe
9.6.1.2) segundo CTe
9.6.1.3) terceiro MDFe

9.7) demais documentos?

9.8) inventar frescuras visuais, deixem isso pro final, precisamos da versão 1.0 primeiro (mapear os inputs e retornos para/da api REST, basicamente)

**Comment by rspadim** [comment]
minha opnião: não precisa se preocupar em fazer uma interface com ajax q nunca 'atualiza a pagina', este tipo de software dificilmente vai rodar em modo offline, pois a interligação depois do offline (html5 offline) é complexa e chata, vamos nos atentar primeiro a fazer ele 'online' (dependente do backend)

**Comment by petersonbrand** [comment]
Seria bom adotar um template padrão pra começar
Tem um que eu sempre vejo o pessoal usar, com bastante colaboradores, é o SB Admin 2

- Bootstrap
- Jquery
- Utiliza o bower
- Tem menu pronto e algumas páginas

https://github.com/BlackrockDigital/startbootstrap-sb-admin-2
http://blackrockdigital.github.io/startbootstrap-sb-admin-2

**Comment by GiseleLima** [comment]
Estou fazendo o layout inicial, espero até inicio da próxima semana disponibilizar para todos

**Comment by JefersonSales** [comment]
Concordo com o petersonbrand  eu uso em projetos aqui esse template ele é bem completo. Mas é bom esperar o que a Gisele tem para mostrar.
